walk-in-interview/src/main/java/com/google/account/FirebaseAuthSessionCookieUtil.java

```diff
@@ -0,0 +1,125 @@
+package com.google.account;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.firebase.auth.SessionCookieOptions;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.CookieParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.URI;
+import java.util.concurrent.TimeUnit;
+
+/** Util methods related to firebase auth session cookies. */
+public final class FirebaseAuthSessionCookieUtil {
+    private static final String ID_TOKEN_PARAM = "idToken";
+    private static final String LOG_IN_PAGE_PATH = "/log-in/index.html";
+
+    public FirebaseAuthSessionCookieUtil() throws IOException {
+        initAdminSDK();
+    }
+
+    /**
+     * Creates the session cookies. It is evoked when the user signs in.
+     *
+     * @param request Log in request.
+     * @return Response with session cookie.
+     */
+    @POST
+    @Path("/business-log-in")
+    @Consumes("application/json")
+    public Response createSessionCookie(HttpServletRequest request) {
+        try {
+            // Gets the ID token sent by the client
+            String idToken = getIDToken(request);
```

Comment:
nit: maybe call this function parseIDToken? just to be consistent with the other servlets